### DISCUSSION TITLE
What is the difference between an "on hand" tool and a not "on hand" tool?

### DISCUSSION THREAD

**Discussion by burgerga** [mealie-recipes/mealie#4498] [Q&A] [answered]
English is not my native language, and I understand what "veggies you have on hand" means. But in context of tools I'm a bit confused... Also looking at https://schema.org/Recipe and asking chatgpt couldn't help me. Is it an optional tool? Is it a category of tools? Can someone give me an example maybe when to use this?

**Comment by Kuchenpirat** [comment, marked as answer]
This is pretty much just a checkbox to mark whether you own that tool or not. 
I don't think we do much if anything with that information, but it might be nice to have that marked for you.

**Reply by burgerga** [reply]
Ahh that makes sense, thanks :)
